tests/compas/files/test_base_reader.py

```diff
@@ -0,0 +1,21 @@
+import os
+try:
+    from pathlib import Path
+except ImportError:
+    from pathlib2 import Path
```

Comment:
is this necessary?

Reply:
Not yet but I planned to give it path objects in tests.

Reply:
Planning to add tests using path objects.